### PR TITLE
ecpdap: init at 0.1.5

### PR DESCRIPTION
**Pull request by expipiplus1**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by legendofmiracles** [comment]
Result of `nixpkgs-review pr 121130` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ecpdap</li>
  </ul>
</details>

File: pkgs/development/tools/ecpdap/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  # This patch is the lock file after running `cargo update`
+  cargoPatches = [ ./lock-update.patch ];
+
+  cargoSha256 = "08xcnvbxm508v03b3hmz71mpa3yd8lamvazxivp6qsv46ri163mn";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ libusb1 ];
+
+  # doCheck = false;
```
Comment (mkg20001) [inline pkgs/development/tools/ecpdap/default.nix:25]:
```suggestion

```

File: pkgs/development/tools/ecpdap/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  # cargoBuildFlags = "--offline";
```
Comment (mkg20001) [inline pkgs/development/tools/ecpdap/default.nix:26]:
```suggestion

```

**Review by mkg20001 — APPROVED** [review]

File: pkgs/development/tools/ecpdap/default.nix (hunk 2 above)
Comment (expipiplus1) [reply pkgs/development/tools/ecpdap/default.nix:26]: oops, how careless of me!

**Pull request merged**